backends-velox/src/test/scala/io/glutenproject/execution/VeloxColumnarCacheSuite.scala

```diff
@@ -110,18 +110,20 @@ class VeloxColumnarCacheSuite extends VeloxWholeStageTransformerSuite with Adapt
   }
 
   test("Support transform count(1) with table cache") {
-    val cached = spark.table("lineitem").cache()
-    try {
-      val df = spark.sql("SELECT COUNT(*) FROM lineitem")
-      checkAnswer(df, Row(60175))
-      assert(
-        find(df.queryExecution.executedPlan) {
-          case _: RowToVeloxColumnarExec => true
-          case _ => false
-        }.isEmpty
-      )
-    } finally {
-      cached.unpersist()
+    withSQLConf(GlutenConfig.COLUMNAR_TABLE_CACHE_ENABLED.key -> "true") {
```

Comment:
we should add this config for the `sparkConf`, it is not a runtime config.

Reply:
I note `buildConf` is used for this property in GlutenConfig, so it should already be a runtime dynamic config now. No?

Reply:
Just figured out it. Please ignore my comment.